soc/nordic/nrf54h/bicr/bicrgen-schema.json

```diff
@@ -0,0 +1,420 @@
+                            "tempDeltaCalibrationTriggerCelsius": {
+                              "type": "number",
+                              "title": "Temperature delta that should trigger calibration (°C)",
+                              "minimum": 0.25,
+                              "maximum": 31.75,
+                              "multipleOf": 0.25,
+                              "default": 1
+                            },
+                            "maxMeasIntervalBetweenCalibrations": {
+                              "type": "number",
+                              "title": "Maximum number measurement intervals between calibrations",
+                              "minimum": 0,
+                              "maximum": 31,
+                              "multipleOf": 1,
+                              "default": 0
+                            }
+                          },
+                          "required": [
+                            "tempMeasIntervalSeconds",
+                            "tempDeltaCalibrationTriggerCelsius",
+                            "maxMeasIntervalBetweenCalibrations"
+                          ]
+                        }
+                      ]
+                    }
+                  }
+                }
+              }
+            }
+          ]
+        }
+      }
+    },
+    "hfxo": {
+      "type": "object",
+      "title": "HFXO Configuration",
+      "properties": {
+        "mode": {
+          "type": "string",
+          "title": "Mode",
+          "enumNames": [
+            "Crystal",
+            "External square signal"
+          ],
+          "enum": [
+            "CRYSTAL",
+            "EXT_SQUARE"
+          ],
+          "default": "CRYSTAL"
+        },
+        "startupTimeUs": {
+          "type": "number",
+          "title": "Startup time (us)",
+          "minimum": 0,
+          "maximum": 4294967294,
+          "default": 850
+        }
+      },
+      "required": [
+        "mode",
+        "startupTimeUs"
+      ],
+      "dependencies": {
+        "mode": {
+          "oneOf": [
+            {
+              "properties": {
+                "mode": {
+                  "const": "CRYSTAL"
+                },
+                "builtInLoadCapacitors": {
+                  "type": "boolean",
+                  "title": "Use built-in load capacitors",
+                  "default": true
+                }
+              },
+              "required": [
+                "builtInLoadCapacitors"
+              ],
+              "dependencies": {
+                "builtInLoadCapacitors": {
+                  "oneOf": [
+                    {
+                      "properties": {
+                        "builtInLoadCapacitors": {
+                          "const": true
+                        },
+                        "builtInLoadCapacitancePf": {
+                          "type": "number",
+                          "title": "Built-in load capacitance (pF)",
```

Comment:
```suggestion
                          "title": "Built-in load capacitance",
```

Reply:
units need to be kept

Reply:
Units are not provided in other UI entries, and they are visible next to the slider, which matches other sliders:
![image](https://github.com/user-attachments/assets/32a8b635-6294-4a80-8a28-dd3d4373c78e)